lib/config-loader.js

```diff
@@ -47,30 +46,23 @@ module.exports.load = (configFilePath, cwd, overrides) => {
 
 function loadConfig (cwd) {
   for (const source of configSources) {
-    let foundPath
+    const foundPath = escalade(cwd, (dir, names) => source && names.includes(source))
+    if (!foundPath) continue
```

Comment:
This needs double checking because initially I used `return` but I believe it was wrong.